app/schemas.js

```diff
@@ -3,5 +3,9 @@
 import { fromJS } from 'immutable';
 
 export default fromJS([
+  // The "getting started" schema provides table-of-contents links
+  // for the sections in src/client/introduction.js.  You are free
+  // to customize or remove both the schema and the introduction component.
+  require('./getting-started.json'),
```

Comment:
I'd rather not, at least for now.  Everything else is done as a separate schema and I'd like to keep this similar, unless/until we revamp things in a way that provides more flexibility for app-specific content.  At this stage, I think it's better to follow the established pattern.